corehq/apps/app_manager/suite_xml/post_process/endpoints.py

```diff
@@ -77,20 +75,20 @@ def _make_session_endpoint(self, module, form=None):
             stack=stack,
         )
 
-    def get_argument_ids(self, frame_children, form=None):
+    def get_argument_ids(self, frame_children, form = None, add_last_selection_datum = True):
```

Comment:
Guessing our linters will complain about this:
```suggestion
    def get_argument_ids(self, frame_children, form=None, add_last_selection_datum=True):
```

Reply:
Made the change as part of https://github.com/dimagi/commcare-hq/pull/33183/commits/d88c99a0b6f9db5d8979d5398143700ef3cb34d7